src/ontology/mondo-edit.obo

```diff
@@ -205423,7 +205434,8 @@ xref: OMIM:250100 {source="MONDO:equivalentTo", source="GARD:0003230"}
 xref: Orphanet:309263 {source="MONDO:equivalentTo"}
 xref: Orphanet:512 {source="OMIM:250100", source="GARD:0003230"}
 xref: Orphanet:751 {source="OMIM:250100"}
-xref: SCTID:238031009 {source="MONDO:equivalentTo"}
+xref: SCTID:238031009 {source="https://orcid.org/0009-0002-1597-2198"}
```

Comment:
Change back to `source="MONDO:equivalentTo"`

Reply:
fixed

Reply:
Actually `source="MONDO:equivalentTo"` needs to be removed to fix the proxy merge issue.